### PR TITLE
add redirect for roles index

### PR DESCRIPTION
**Pull request by leina05**
Currently the old link to the roles getting started index is broken (linked to from live blog posts).

### CODE REVIEW

File: new-docs/content/any/getting-started/roles/_index.md [hunk 1]
```diff
@@ -4,6 +4,8 @@ weight: 3
 description: >
   Roles are an intuitive way to group permissions and expose them to end
   users. Use Oso to add roles to your application.
+aliases:
+  - getting-started/roles/index.html
```
Comment (samscott89) [inline new-docs/content/any/getting-started/roles/_index.md:8]: Have you tested this locally? The aliases often need to be relative paths or it'll override it to one of the language-specific guides.

We also have a clashing link here: https://github.com/osohq/oso/blame/main/new-docs/content/python/getting-started/roles/builtin_roles.md#L5

Which may or may not be fine, I'm not sure.

Comment (leina05) [reply new-docs/content/any/getting-started/roles/_index.md]: yes good catch, tested locally and fixed the duplicate, thanks! Should be good now. It is weird how the aliases work with the language selector...

File: new-docs/content/python/getting-started/roles/builtin_roles.md [hunk 2]
```diff
@@ -42,15 +39,15 @@ relationships you can query, as well as `User.widget_roles` and `Widget.roles`
...
-  The names of the relationships we add to models are generated by the
-  following conventions:
-
-  * `Widget.users`: this relationship is always called `users`, regardless of
-    the actual user model name
-  * `User.widgets`: `Widget.__name__ + "s"`
-  * `Widget.roles`: this relationship is always called `roles`
-  * `User.widget_roles`: `Widget.__name__.lower() + "_roles"`
-{{% /callout %}}
+The names of the relationships we add to models are generated by the
```
Comment (samscott89) [inline new-docs/content/python/getting-started/roles/builtin_roles.md:42]: We're going to need to standardise on markdown formatters. Mine and Gabe's are incompatible too.

**Review by samscott89 — APPROVED** [review]

File: new-docs/content/python/getting-started/roles/builtin_roles.md (hunk 2 above)
Comment (leina05) [reply new-docs/content/python/getting-started/roles/builtin_roles.md:42]: Ugh yes forgot about this

**Pull request merged**